api/external_resources/models.py

```diff
@@ -0,0 +1,102 @@
+from django.db import models
+from django_lifecycle import (
+    AFTER_SAVE,
+    BEFORE_DELETE,
+    LifecycleModelMixin,
+    hook,
+)
+
+from features.models import Feature, FeatureState
+from integrations.github.tasks import call_github_app_webhook_for_feature_state
+from webhooks.webhooks import WebhookEventType
+
+
+class ExternalResources(LifecycleModelMixin, models.Model):
```

Comment:
It is any reference to an external resource using a URI. In this case it is used to link a GH PR/Issue to a flag. You can use it in the future to link any kind of external resource like a JIRA ticket, slack comment, etc. Once we have more than one external App, we could use Django Generic Relations and ContentType to better model this external-resources and its types.

Reply:
This has been discussed with Matt and for now, this is generic enough considering the only additional case of use for external resources which is JIRA integration. In the future, it could be necessary to extend it as discussed.